mod/wikirate/set/self/recent_contributor.rb

```diff
@@ -0,0 +1,8 @@
+
+def item_names args={}
+  limit = Env::params[:limit] || 5
+  Card::Act.select(:actor_id).distinct.where("card_id is not NULL and actor_id <> #{Card["wagn bot"]} and actor_id <> #{Card["Anonymous"]}").order('id DESC').limit(limit).map do |act|
+    Card[act.actor_id].id
+  end
```

Comment:
no comment, no helpful method names...

at this point we need cleaner code.  a massive query like that needs its own method.

Why is a method named "item_names" returning ids?